themes/default/content/registry/packages/azure-native/installation-configuration.md

```diff
@@ -4,8 +4,13 @@ meta_desc: How to set up credentials to use the Pulumi Azure Native Provider and
 layout: installation
 ---
 
+To provision resources with the Pulumi Azure Native provider, you need to have Azure credentials.
+These instructions assume you're using the [Azure CLI 2.0](https://github.com/Azure/azure-cli).
+Your Azure credentials are never sent to Pulumi.com.
+Pulumi uses the Azure SDK and the credentials in your environment to authenticate requests from your computer to Azure.
+
 {{% notes type="info" %}}
-Pulumi Azure also works with the legacy [Azure xPlat CLI](https://github.com/Azure/azure-xplat-cli) but we recommend using the CLI 2.0 for the best experience.
+Pulumi Azure Native also works with the legacy [Azure xPlat CLI](https://github.com/Azure/azure-xplat-cli) but we recommend using the CLI 2.0 for the best experience.
```

Comment:
I doubt that xPlat still works because some auth code explicitly calls out to `az`. Has anyone tested this in recent years?

Reply:
The last commit on xPlat was five years ago, so I suggest removing it.